### ISSUE TITLE
Error on Participant Upload

### ISSUE THREAD

**Issue by blynchNDI** [nditech/apollo#361]
Uploading a participant set caused an error, and seemed to actually have uploaded, but leads to a 502 error when you try to click manage (https://apolloqa.demcloud.org/participants/set/4)

The docker logs are pretty long, but some interesting lines are: 

2019-05-31T16:54:40.678069838Z [2019-05-31 16:54:40,677] ERROR in app: Exception on /participants/set/4 [GET]

2019-05-31T16:54:40.678124933Z psycopg2.DataError: value "200100100210" is out of range for type integer

2019-05-31T16:54:40.680824930Z jinja2.exceptions.TemplateNotFound: 9h9h.html

**Comment by blynchNDI** [comment]
Below is the participant file that caused the issue (zipped because github doesn't allow csv uploads).

[Nigeria Participants 2019 Test - 20190531.zip](https://github.com/nditech/apollo/files/3242216/Nigeria.Participants.2019.Test.-.20190531.zip)

**Comment by SarahCooperNDI** [comment]
We initially tried the upload as an .xlsx file (below). That upload also failed due to invalid file type. So we would need to address the issues both with the .xlsx and the .csv  This is a direct export from the 2019 WTV database (we just tried to replicate the structure in Apollo 3) so there shouldn't be formatting issues?

[Nigeria Participants 2019 Test - 20190531.xlsx](https://github.com/nditech/apollo/files/3242542/Nigeria.Participants.2019.Test.-.20190531.xlsx)

**Comment by SarahCooperNDI** [comment]
Similarly had issues in the participant set TUNISIA CORRECT ENCODING.  Excel file (attached) refused to upload due to invalid file type.  CSV file (exported from the set Tunisia Participants Test) allowed me to map fields and did not return an error message, but no participants were imported.

[Tunisia Participants Test.xlsx](https://github.com/nditech/apollo/files/3242660/Tunisia.Participants.Test.xlsx)

[tunisia-participants-test-participants-2019-05-31-193016.zip](https://github.com/nditech/apollo/files/3242663/tunisia-participants-test-participants-2019-05-31-193016.zip)

**Comment by SarahCooperNDI** [comment]
Saved the below Excel file as a comma delimited CSV and added UTF-8 encoding under web options.  The CSV still isn't properly reading the Arabic.  Thoughts?

[Tunisia Locations UTF 8 - 20190531.zip](https://github.com/nditech/apollo/files/3242693/Tunisia.Locations.UTF.8.-.20190531.zip)

[Tunisia Locations UTF 8 - 20190531.xlsx](https://github.com/nditech/apollo/files/3242691/Tunisia.Locations.UTF.8.-.20190531.xlsx)

**Comment by blynchNDI** [comment]
For the sake of closing 3.7, I'm moving this out to a future sprint. Still blocked on this from not having a the nigeria location set, but there seems to be some strange behavior happening from trying to upload both the nigeria and tunisia sets. Will try to update at a later point once I have more detailed information on this.

**Comment by blynchNDI** [comment]
Closing this, as I was not able to get the corresponding location sets. Will probably reopen as a new issue if I do get the location sets
